### PR TITLE
[ mission-control ] - Add support to external database secrets

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

-  Add support to external database secrets
-  Add support to use single user/password for all services

### CODE REVIEW

File: stable/mission-control/templates/mission-control-statefulset.yaml [hunk 1]
```diff
@@ -148,13 +148,8 @@ spec:
         - name: PGPASSWORD
           valueFrom:
             secretKeyRef:
-              {{- if .Values.postgresql.postgresqlPassword }}
-              name: {{ template "mission-control.fullname" . }}-db-cred
-              key: postgres-password
-            {{- else }}
               name: {{ .Release.Name }}-postgresql
```
Comment (danielezer) [inline stable/mission-control/templates/mission-control-statefulset.yaml:151]: This won’t work with an external posters server.

File: stable/mission-control/templates/postgresql-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  {{- end }}
+  {{- if .Values.postgresql.db.jfscPassword }}
+  jfscPassword: {{ .Values.postgresql.db.jfscPassword | b64enc | quote }}
+  {{- else }}
+  jfscPassword: {{ randAlphaNum 10 | b64enc | quote }}
+  {{- end }}
+  {{- if .Values.postgresql.db.jfexPassword }}
+  jfexPassword: {{ .Values.postgresql.db.jfexPassword | b64enc | quote }}
+  {{- else }}
+  jfexPassword: {{ randAlphaNum 10 | b64enc | quote }}
```
Comment (danielezer) [inline stable/mission-control/templates/postgresql-secrets.yaml:26]: Why are all of these here if you removed it from the values?

File: stable/mission-control/templates/mission-control-statefulset.yaml (hunk 1 above)
Comment (chukka) [reply stable/mission-control/templates/mission-control-statefulset.yaml:151]: This condition is only for {{- if .Values.postgresql.enabled }}

File: stable/mission-control/templates/postgresql-secrets.yaml (hunk 2 above)
Comment (chukka) [reply stable/mission-control/templates/postgresql-secrets.yaml:26]: These are for internal postgres ~ postgresql.enabled=true, removed only for external database password

**Comment by danielezer** [comment]
It doesn’t make sense in my opinion. It should either be one secret in both cases or all secrets in both cases...

**Comment by chukka** [comment]
@danielezer Updated changes for both internal/external databases

**Pull request merged**